clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/1_node/max_volumes_per_pod/override.yaml

```diff
@@ -1,2 +1,2 @@
 TOTAL_PODS: 1
-VOLUMES_PER_POD: 100
+VOLUMES_PER_POD: 10
```

Comment:
I don't have strong opinion about this one. I would like to avoid having donzens of jobs and growing this matrix of scenarios, but I don't know if we should have 10 or 100 here.

Reply:
@msau42 - WDYT?

Reply:
Realistically I don't think a single pod will have 100 volumes.  5-10 seems more realistic, but the initial purpose of this experiment is to find our current scaling thresholds.  

I think emptydir and downward API volumes performance is acceptable.  Unsure about secrets and configmaps.  Again I want to find the threshold where performance starts to fall off.

Reply:
> I think emptydir and downward API volumes performance is acceptable. Unsure about secrets and configmaps. Again I want to find the threshold where performance starts to fall off.

I think at least part of the problem is in QPS limits - we need to GET each secret/configmap from apiserer before we even start the pod. So with 5qps limit, 100 volumes just means 20s at start.

I think that what we should seriously consider (at least as an experiment) is to bump it to something large (say 100+) and see what will happen. I would even say that it's probably first thing to do.

Reply:
Is it meaningful to add the wait time for configmap and secret volumes be available to pod startup time? Should we add some barrier to wait they are available before starting to pod creation?